### PR TITLE
Hooked up player list on front end

### PR DESCRIPTION
**Pull request by michaelli00**

### CODE REVIEW

File: server/db.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-const Pool = require('pg').Pool;
+const { Pool } = require('pg');
 
 const pool = new Pool({
     user: 'postgres',
     password: 'postgres',
     host: 'localhost',
     port: 5433,
-    database: 'league'
+    // database: 'league'
```
Comment (michaelli00) [inline server/db.js:8]: I had to remove this line locally to make the db work

File: server/index.js [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 
+// TODO why can't we just return an array rather than access it through .list field
 // returns the id, name and rating of each player
 // returns { "list" : [ {"pid" : player_ID, "pname" : player_name, "pr" : rating, "active" : active}, ... ] }
 app.get('/api/players', (req, res) => {
     pool.query("select player_id as pid, player_name as pname, rating as pr, active from ratings order by rating desc, player_id asc", (err, results) => {
         // console.log("results:\n"+JSON.stringify(results.rows)+"\n\n\n");
         if (err) { /*console.log("hey there was an error: \n" + err);*/ res.status(500).send("something went wrong"); return;}
-        let result = {"list": results.rows}
+        let result = results.rows;
```
Comment (michaelli00) [inline server/index.js:22]: Any reason why we can't just return `results.rows`?

File: server/db.js (hunk 1 above)
Comment (yashkaps) [reply server/db.js:8]: if u remove that line then u won't be connecting to the right database. it prob didn't work for u cuz u need to open postgres on ur side and create the database and the tables according to the db.sql file

File: server/index.js (hunk 2 above)
Comment (yashkaps) [reply server/index.js:22]: i read somewhere online that it's bad practice to directly return an array and that the highest level of what i'm returning should be able to be parsed as a dictionary/hashmap. and if i'm just returning a list then i should have it as a field.

also it makes it easier for us to add other stuff later on if we need to. for example returning an array of objects immediately fails if i also need to return the number of objects that is returned. but in the way that i was doing it, it'd be as simple as just adding another field to the object, and the rest of ur code would not need to be changed in any way to account for this change.

**Pull request closed without merging**